deploy/olm-catalog/ibm-block-csi-operator/1.1.0/ibm-block-csi-operator.v1.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,572 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: ibm-block-csi-operator.v1.1.0
+  namespace: placeholder
+  annotations:
+    capabilities: "Seamless Upgrades"
+    categories: "Storage,Cloud Provider"
+    certified: "true"
+    containerImage: registry.connect.redhat.com/ibm/ibm-block-csi-operator:1.1.0
```

Comment:
Should we move to [registry.redhat.io](https://access.redhat.com/RegistryAuthentication#setting-registryredhatio-as-the-default-8)?

Reply:
Let's update this field when we push the image or we are going to do it.

Reply:
OK